bot_env.py

```diff
@@ -3,15 +3,16 @@
 from math import inf, ceil
 from typing import Optional
 
-PAN_LOW = 350
-PAN_HIGH = 700
-TILT_LOW = 475
-TILT_HIGH = 725
+PAN_LOW = -35
```

Comment:
Can we keep the old robot code? I still want to combine old data with new data.

Reply:
Conversation note: we could make a separate goal to predict other robot telemetry from video stream only. Might be handy to train on random Internet videos.